examples/cluster_groups.py

```diff
@@ -75,3 +75,13 @@
 cluster_group = cluster_groups.get_by_name(cluster_group_name)
 print(f"{cluster_group}")
 print(f"{pp.pformat(cluster_group.data)} \n")
+
+cluster_group_name = cluster_group_object.data["name"]
+print("\n\nrename")
+cluster_group = cluster_group_object
```

Comment:
Do we need to assign cluster object to a different variable?

Reply:
It doesn't, it was just to keep the code looking the same from all the other steps.  

I removed the line and updated the references below to cluster_group